src/map/unit.cpp

```diff
@@ -4151,8 +4148,6 @@ int32 unit_free(struct block_list *bl, clr_type clrtype)
 
 			skill_clear_unitgroup(bl);
 			status_change_clear(bl,1);
-
-			ed->~s_elemental_data();
```

Comment:
So the previous commit was wrong? Maybe should link to it: https://github.com/rathena/rathena/commit/273d1034251ce8a8d25f3ce0d1a24f50b1b8e955

Reply:
It was not wrong.
All the destructor calls should have been moved in c198bb3
But I missed this one.
The call to the destructor is still there and now happens in map_destroyblock